spec/models/candidate_interface/personal_details_form_spec.rb

```diff
@@ -124,8 +124,26 @@
     it { is_expected.to validate_length_of(:first_name).is_at_most(100) }
     it { is_expected.to validate_length_of(:last_name).is_at_most(100) }
 
-    it { is_expected.to validate_inclusion_of(:first_nationality).in_array(NATIONALITY_DEMONYMS) }
-    it { is_expected.to validate_inclusion_of(:second_nationality).in_array(NATIONALITY_DEMONYMS) }
+    it 'validates nationalities against the NATIONALITY_DEMONYMS list' do
+      details_with_invalid_nationality = CandidateInterface::PersonalDetailsForm.new(
+        first_nationality: 'Tralfamadorian',
+        second_nationality: 'Czechoslovakian',
+      )
+
+      details_with_valid_nationality = CandidateInterface::PersonalDetailsForm.new(
+        first_nationality: NATIONALITY_DEMONYMS.sample,
+        second_nationality: NATIONALITY_DEMONYMS.sample,
+      )
+
+      details_with_valid_nationality.validate
+      details_with_invalid_nationality.validate
+
```

Comment:
Is it worth adding?:

```
expect(details_with_valid_nationality).to be_valid
expect(details_with_invalid_nationality).not_to be_valid
```

Reply:
I ended up not doing this because there might be other properties of the object which affect validity. If we had a factory for a personal_details_form we might `build` one but that seems ott?

Reply:
makes sense, come to think of it tests that rely on other attributes all being valid can be brittle so best as is I think